docker-compose.yml

```diff
@@ -108,6 +108,19 @@ services:
     networks:
       - primary
 
+  dbmate:
```

Comment:
Use our own image to run them. For local development, you can run the migration locally in bash.